src/components/EnterpriseList/EnterpriseList.test.jsx

```diff
@@ -116,76 +158,24 @@ describe('<EnterpriseList />', () => {
         }],
         start: 0,
       };
+
       wrapper = mount((
         <MemoryRouter initialEntries={['/test']}>
-          <EnterpriseList
-            enterprises={oneEnterpriseListData}
-            getEnterpriseList={() => {}}
-            clearPortalConfiguration={() => {}}
-            getLocalUser={() => {}}
-            setSearchQuery={() => {}}
-          />
+          <ContextProvider>
```

Comment:
Similar comment as above regarding just killing `ContextProvider` and going with `<Provider>`.